scranShipper/src/Basket/BasketItem/index.jsx

```diff
@@ -0,0 +1,28 @@
+
+
+
+
+const BasketItem = (props) => {
+
+    const {updateOrder} = props
```

Comment:
On several lines you're not using semicolons, but you are in other places. It doesn't matter which style you do, just pick one and stick to it!